specificatie/datum.yaml

```diff
@@ -0,0 +1,163 @@
+openapi: 3.0.3
+info:
+  title: Datum definities
+  description: |
+    Definities voor de volgende datum types:
+    - datum conform LO GBA
+    - onbekend datum
+    - 'volledig' datum conform ISO-8601
+    - datum waarvan de dag onbekend is
+    - datum waarvan de maand en dag onbekend is 
+  version: 2.0.0
+  contact: {}
+paths:
+  /examples/datums:
+    get:
+      responses:
+        '200':
+          description: |
+            Als voorbeeld een lijst met LO GBA datum soorten
+
+            - volledig
+            - onvolledig: dag onbekend
+            - onvolledig: maand en dag onbekend
+            - onvolledig: volledig onbekend
+
+            en hun Haal Centraal tegenhanger
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  datums:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/AbstractDatum'
+                example:
+                  [
+                    { "type": "GbaDatum", "waarde": "20180701" },
+                    { "type": "GbaDatum", "waarde": "20161200" },
+                    { "type": "GbaDatum", "waarde": "20100000" },
+                    { "type": "GbaDatum", "waarde": "00000000" },
+                    { "type": "Datum", "waarde": "2018-07-01" },
+                    { "type": "JaarMaandDatum", "jaar": "2016", "maand": "12" },
+                    { "type": "JaarDatum", "jaar": "2010" },
+                    { "type": "OnbekendDatum", "onbekend": true }
+                  ]
+components:
+  schemas:
+    AbstractDatum:
+      type: object
+      required:
+        - type
+      properties:
+        type:
+          type: string
+      discriminator:
+        propertyName: type
+        mapping:
+          GbaDatum: '#/components/schemas/GbaDatum'
+          Datum: '#/components/schemas/Datum'
+          OnbekendDatum: '#/components/schemas/OnbekendDatum'
+          JaarDatum: '#/components/schemas/JaarDatum'
+          JaarMaandDatum: '#/components/schemas/JaarMaandDatum'
+    GbaDatum:
+      description: Datum conform LO GBA
+      allOf:
+        - $ref: '#/components/schemas/AbstractDatum'
+        - type: object
+          properties:
+            waarde:
+              type: string
+              pattern: "^[0-9]{8}$"
+              minLength: 8
```

Comment:
waarom hier ook minLength en maxLength? De lengte zit al in het paters

Reply:
fixed